src/components/document/student-group-view.scss

```diff
@@ -0,0 +1,22 @@
+@import "../vars.sass";
+
+.student-group-view {
+  .canvas-area {
+    position: absolute;
+    top: $workspace-titlebar-height;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    border-top: $document-border-width solid white;
+  }
+}
+
+.nav-tab-panel {
+  .student-group-view {
+    top: $workspace-titlebar-height;
+    margin: 0;
+    padding-right: 3px;
+    border: 1.5px solid $workspace-teal;
+    border-radius: 0;
+  }
+}
```

Comment:
A general question when adding new style files: do we prefer .sass or .scss at this point?  If one of preferred for this project, why?

Reply:
I think we should be moving towards `.scss` -- `.sass` was the original format and `.scss` is essentially the version 2 format. I prefer it for several reasons:
1. it's the more modern format
2. it's more compatible with `.css` (and `.less`) files
  - this came up when converting the custom-select which used a `.less` file
3. it's required to support the import of sass variables into JavaScript/TypeScript